deskcon.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 
-BINPATH=`dirname $0`
-cd $BINPATH/server/
-./deskcon-server
\ No newline at end of file
+BINPATH=$(pwd)
+$BINPATH/server/deskcon-server.py
```

Comment:
was asking myself the same question. i just noticed that this script could have never worked as stated in the readme. there is no practical reason that this file is needed anyway. 
a better aproach would be to state in the documentation that `server/deskcon-server.py`should be called.

I will provide a systemd service for deskcon-server.py and a Makefile in the next few days when i get home